test/unit/EmptyState.spec.js

```diff
@@ -0,0 +1,55 @@
+import { mount, createLocalVue } from '@vue/test-utils';
```

Comment:
It would be great if you could create a test to check if the event `add-button-click` is emited properly when the button is clicked